views/manager/_form.php

```diff
@@ -19,12 +19,15 @@
 echo $form->field($model, 'published')->checkbox();
 
 $settings = [
-    'lang' => Yii::$app->language,
     'minHeight' => 200,
     'plugins' => [
         'fullscreen',
     ],
 ];
+if (!in_array(Yii::$app->language, ['en', 'en_US', 'en-US'])) {
```

Comment:
This all languages of Imperavi: https://github.com/vova07/yii2-imperavi-widget/tree/master/src/assets/lang And there en_GB is no =)